tools/files/tests.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - stable-*
+    tags:
```

Comment:
We don't currently run any of these workflows when a release is tagged, only the release workflows. Is there a reason to add this?